docs/src/pages/components/masonry/ImageMasonry.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function ImageMasonry() {
+  return (
+    <Masonry
+      columns={{ xs: 3, sm: 5, md: 7 }}
+      spacing={{ xs: 1, sm: 2, md: 3 }}
+      sx={{ height: 300 }}
+    >
+      {itemData.map((item) => (
+        <MasonryItem key={item.img}>
+          <img
+            srcSet={`${item.img}?fit=crop&auto=format 1x,
+                ${item.img}?fit=crop&auto=format&dpr=2 2x`}
+            alt={item.title}
+            loading="lazy"
+          />
+        </MasonryItem>
+      ))}
+    </Masonry>
+  );
+}
+
+const itemData = [
+  {
+    img: 'https://images.unsplash.com/photo-1518756131217-31eb79b20e8f',
+    title: 'Fern',
+  },
+  {
+    img: 'https://images.unsplash.com/photo-1597645587822-e99fa5d45d25',
+    title: 'Mushrooms',
+  },
+  {
+    img: 'https://images.unsplash.com/photo-1529655683826-aba9b3e77383?ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8bG9uZG9ufGVufDB8fDB8fA%3D%3D&ixlib=rb-1.2.1&w=1000&q=80',
+    title: 'Tower',
+  },
+  {
+    img: 'https://images.unsplash.com/photo-1471357674240-e1a485acb3e1',
+    title: 'Sea star',
+  },
+  {
+    img: 'https://images.pexels.com/photos/1761279/pexels-photo-1761279.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
```

Comment:
Perhaps we should use a single source for all the images?

Reply:
Okay, I will keep those from images.unsplash.com!